lib/rotoscope.rb

```diff
@@ -70,4 +80,46 @@ def short_log_path
     chars = log_path.chars
     "#{chars.first(17).join('')}...#{chars.last(20).join('')}"
   end
+
+  def log_call
+    return if blacklist.match?(caller_path)
+
+    if caller_method_name.nil?
+      caller_class_name = '<ROOT>'
+      caller_method_name = '<UNKNOWN>'
+      caller_method_level = '<UNKNOWN>'
+    else
+      caller_class_name = self.caller_class_name
+      caller_method_name = escape_csv_string(self.caller_method_name)
+      caller_method_level = caller_singleton_method? ? 'class' : 'instance'
+    end
+
+    call_method_level = singleton_method? ? 'class' : 'instance'
+    method_name = escape_csv_string(self.method_name)
+
+    io <<
+      '"' << class_name << '",' \
```

Comment:
I know this was already like this, but why do we need to wrap the values inside quotes?

Reply:
The quotes allow more characters to be used without escaping (e.g. spaces, commas).  I don't think this is necessary for the class name.  The file path isn't as restricted as ruby constants, so theoretically should also be escaped with escape_csv_string.  The method name for test method has caused us problems already, which is why we have added `escape_csv_string` to properly escape it.

Reply:
👍 , my question was more about the `class_name`, just wanted to make sure I wasn't missing something. Doesn't heart to escape it even if it's not needed

Reply:
lol "doesn't **heart**" 🤦‍♂️